model.py

```diff
@@ -278,8 +279,20 @@ def from_scraper(cls, type, scraper, alias_to_id_map, region_id):
 
 class PendingTournament(object):
     '''Same as a Tournament, except it uses aliases for players instead of ids.
-       Used during tournament import, before alias are mapped to player ids.'''
-    def __init__(self, type, raw, date, name, players, matches, regions, alias_to_id_map=[], id=None):
+       Used during tournament import, before aliases are mapped to player ids.'''
+    def __init__(self, type, raw, date, name, players, matches, regions, alias_to_id_map=None, id=None):
```

Comment:
Note to self, check rest of codebase for other named arguments where I default to list (i.e. alias_to_id_map=[]).